vdev/src/testing/config.rs

```diff
@@ -108,6 +108,8 @@ pub struct IntegrationTestConfig {
     pub test: Option<String>,
 
     pub test_filter: Option<String>,
+
+    pub paths: Option<Vec<String>>,
```

Comment:
Is it really optional at this point? If not, you could drop the `Option`.

Reply:
The only one that doesn't have paths defined is `shutdown` , which seemed reasonable to leave as merge queue only.